### ISSUE TITLE
The Compose file...is invalid because...

### ISSUE THREAD

**Issue by davidshq** [wordup-dev/wordup-cli#13]
Tried to run:

wordup wpcli plugin install cmb2

Received the following response

--- Currently running wordup projects: liquid-messages ---

Run command: wp plugin install cmb2
The Compose file 'C:\Users\david\AppData\Roaming\npm\node_modules\wordup-cli\docker\docker-compose.yml' is invalid because:
services.mail.ports contains an invalid type, it should be a number, or an object
services.wordpress.ports contains an invalid type, it should be a number, or an object

**Comment by shry** [comment]
I investigated this issue, and i came to the conclusion that the downgrade of the docker-compose version in 0.7.3 is not compatible with Windows. 
If you change the version (at the beginning of the file) in 'C:\Users\david\AppData\Roaming\npm\node_modules\wordup-cli\docker\docker-compose.yml' back to 3.7, does it work again?

**Comment by ddazal** [comment]
Hi, I was having the same issue and I "solved" it by hardcoding the ports in the `docker-compose.yml` file. I realize it's not the best solution but is working for my needs though. The version of the docker-compose file is 3.3.

```yml
services:
  wordpress:
    ports:
      - "8000:80"
 mail:
    ports:
      - "8025:8025"
```

**Comment by shry** [comment]
> Hi, I was having the same issue and I "solved" it by hardcoding the ports in the `docker-compose.yml` file. I realize it's not the best solution but is working for my needs though. The version of the docker-compose file is 3.3.
> 
> ```yaml
> services:
>   wordpress:
>     ports:
>       - "8000:80"
>  mail:
>     ports:
>       - "8025:8025"
> ```

Thank you for your help. Have you tried also my idea of changing the docker-compose version to 3.7? Like i wrote in:  https://github.com/wordup-dev/wordup-cli/issues/13#issuecomment-526604793  
I still think that this causes the bug.

**Comment by ddazal** [comment]
Yes, I tried that with no success. I'm using Ubuntu 16 an the error is the same :(
```
services.mail.ports contains an invalid type, it should be a number, or an object
services.wordpress.ports contains an invalid type, it should be a number, or an object

```

**Comment by shry** [comment]
Ok. Thank you. I'm currently rewriting/refactoring a lot of parts of the wordup-cli, perhaps this bug will be fixed afterwards ;)
